examples/data/json-units/benchmark_production_OECM.json

```diff
@@ -2,7 +2,7 @@
   "benchmark_temperature": "1.5 delta_degC",
   "benchmark_global_budget": "396 Gt CO2",
   "is_AFOLU_included": false,
```

Comment:
S1S2S3 is 'AllScopes'.  Also the TPI benchmarks are based on fully blended S1S2S3 numbers.  TPI has a different global budget, different numbers, and different decarbonization pathways, but an internally consistent logic that we can follow and use.  When a company reports S1, S2, and (categorized) S3 numbers, the tool can blend them to get a valid calc.  Alas, there's no way to judge whether the S1+S2 components are heavy or light with respect to the S3 component, but it should still be possible to show how the two do stack up against the S1S2S3 number.

Reply:
what do you suggest me to change?

Reply:
Suggesting you change 'AllScopes' to 'S1S2S3' both in the code and the benchmark.  The OECM data processing in itr-data-pipeline should be able to generate a proper S1S2S3 benchmark (which would save you the trouble of creating it from two separate benchmarks).  I'll be AFK for an hour, then can answer back.

Reply:
We earlier discussed, that production bm are unrelated to scope.
"AllScopes" is an attempt to highlight this fact.
Changing to "S1S2S3" may be confusing, because it's a standalone "scope" within our logic across the tool.
Let's find a more suitable name for this item.

"ScopeIndependent"? "General"? "All"?

Reply:
I renamed into "AnyScope".
This might be more clear